Sources/TuistKit/Utils/TextTable.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+
+struct TextTable<T> {
```

Comment:
Considering this is not Tuist-specific, what about moving it to `TuistSupport`? I really like the interface of this one btw :)

Reply:
+1 - I'd also add some simple documentation for it if we move it to `TuistSupport`

Reply:
Sure! I moved the `TextTable` to `TuistSupport` and added the documentation ✔️